docs/source/Usage.md

```diff
@@ -136,6 +136,9 @@ The following is the list of the available analyzers you can run out-of-the-box:
 * `DNStwist`: Scan a url/domain to find potentially malicious permutations via dns fuzzing. [dnstwist repo](https://github.com/elceef/dnstwist) 
 * `IPInfo`: Location Information about an IP
 * `Zoomeye`: [Zoomeye](https://www.zoomeye.org) Cyberspace Search Engine recording information of devices, websites, services and components etc..
+* `Inquest_IOCdb`: Indicators of Compromise Database - search IOCdb
```

Comment:
same typo here...

Reply:
also, people could not know what InQuest is. Please provide a link like it was done in other descriptions

Reply:
Link for all 3 analyzers?